Gemfile

```diff
@@ -27,6 +27,13 @@ gem 'bootsnap', '>= 1.4.4', require: false
 # Use Rack CORS for handling Cross-Origin Resource Sharing (CORS), making cross-origin AJAX possible
 # gem 'rack-cors'
 
+# Sentry error monitoring
+gem 'sentry-ruby', '~> 4.6.4'
+gem 'sentry-rails', '~> 4.6.4'
+
+# Set Rails-only ENV with Figaro
+gem 'figaro', '~> 1.2'
```

Comment:
Whoa. Haven't seen figaro for a long time :D While it's probably still fine, most people use [dotenv](https://github.com/bkeepers/dotenv) this days or just [Rails credentials](https://edgeguides.rubyonrails.org/security.html#environmental-security)

Reply:
Noted! Figaro was the first thing I found, and last version came out _just over a year ago_, so I've decided it's fine. I'll leave it for now and if needed I'll upgrade to one of the above.